deploy/004-deploy-mock-pools.ts

```diff
@@ -116,13 +101,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     collateralFactor: convertToUnit(0.7, 18),
     liquidationThreshold: convertToUnit(0.7, 18),
     accessControlManager: accessControlManager.address,
-    vTokenProxyAdmin: deployer,
-    tokenImplementation_: tokenImplementation.address,
+    vTokenProxyAdmin: proxyAdmin,
+    tokenImplementation_: vBSWImplementation.address,
   });
-  await tx.wait();
+
+  console.log("Pools added to pool: " + comptroller1Proxy.address);
 };
 
 func.tags = ["Pools"];
-func.dependencies = ["PoolsRegistry"];
```

Comment:
Can we add a skip property on this so it only runs on say local environments?

Reply:
yeah